dojo/finding/views.py

```diff
@@ -1696,7 +1696,7 @@ def request_finding_review(request, fid):
                 jira_helper.push_to_jira(finding.finding_group)
 
             reviewers = Dojo_User.objects.filter(id__in=form.cleaned_data["reviewers"])
-            reviewers_string = ", ".join([str(user) for user in reviewers])
+            reviewers_string = f"({user.id}), ".join([str(user) for user in reviewers])
```

Comment:
Is this working? Because I suppose `join` has no information about the user. Might be better:

```suggestion
            reviewers_string = ", ".join([f"{user} ({user.id})" for user in reviewers])```